repository/src/main/java/org/apache/atlas/repository/store/graph/v2/AtlasStructDefStoreV2.java

```diff
@@ -407,9 +407,6 @@ public static void updateVertexPreUpdate(AtlasStructDef structDef, AtlasStructTy
                         removedAttributes.add(currAttrName);
 
                         LOG.warn("REMOVED ATTRIBUTE: {}.{}", structDef.getName(), currAttrName);
-                    } else {
```

Comment:
@GhalebKhaled, Can you add condition for only BM type attributes only ? so that other type attributes are not deleted accidentally.